examples/service-account.go

```diff
@@ -44,8 +45,10 @@ func main() {
 	ctx := context.Background()
 
 	// add service account
+	expiration := time.Now().Add(30 * time.Minute)
 	addReq := madmin.AddServiceAccountReq{
 		TargetUser: "my-username",
+		Expiration: &expiration,
```

Comment:
Hi @Praveenrajmani, when I don't send in the `expiration` field here, the API seems to return an `expiration` value of `1970-01-01T00:00:00Z`. Is this expected behavior?

<img width="564" alt="Screen Shot 2023-05-08 at 7 34 46 PM" src="https://user-images.githubusercontent.com/27314853/236979461-eed59210-354c-4027-8272-afb9839a0013.png">

You can reproduce the issue by applying the following diff:

```diff
diff --git a/examples/service-account.go b/examples/service-account.go
index 73856cf..88b02a9 100644
--- a/examples/service-account.go
+++ b/examples/service-account.go
@@ -45,10 +45,8 @@ func main() {
 	ctx := context.Background()
 
 	// add service account
-	expiration := time.Now().Add(30 * time.Minute)
 	addReq := madmin.AddServiceAccountReq{
 		TargetUser: "my-username",
-		Expiration: &expiration,
 	}
 	addRes, err := madminClient.AddServiceAccount(context.Background(), addReq)
 	if err != nil {
```

Reply:
Well that is because it is considered no expiration @kaankabalak